jboss/container/wildfly/launch/datasources/added/launch/datasource-common.sh

```diff
@@ -174,9 +174,8 @@ function inject_external_datasources() {
         db="POSTGRESQL"
       elif [ "$driver" == "mysql" ]; then
         db="MYSQL"
```

Comment:
It seems we can safely remove the calculation of `driver` and `db` variable here in this method. I cannot see how they are used outside of this function. `db` is declared as local variable and it is not used in this function so that one for sure can be removed. I haven't found any external uses of the `driver` variable declared here. What's the point of having them here?

Reply:
@yersan 
I think it is something along the lines of:
* If we do not call map_properties further down, we cannot configure xa-datasources since the xa-connnection-properties do not get calculated from the database name.
* map_properties needs the driver parameter or it does not configure the xa-datasource. Also, it needs to differentiate between postgres and mysql to decide on Port vs PortNumber as the name for the xa-connection property
* We could also probably pass in the env variables for all these connection properties when starting the container, but I am already unsure about the new need to pass in the extra connection-checker and exception-sorter variables that you can see in configure.bats

Reply:
Ok, my error here was to see the locally declared variable as local to the current function, however, functions called from the current one will also see the `db` variable